pylsp_mypy/plugin.py

```diff
@@ -164,6 +166,23 @@ def pylsp_lint(
         return get_diagnostics(config, workspace, document, settings, is_saved)
 
 
+@lru_cache(maxsize=2000)
+def get_venv_cmd(path: str, workspace: str) -> List[str]:
+    if not shutil.which("get-venv.py"):
+        return []
+    try:
+        p = subprocess.run(
+            ["get-venv.py", json.dumps(dict(cmd=["mypy"], path=path, workspace=workspace))],
```

Comment:
in theory, we could send the entire `args` list as well - this would provide maximum flexibility at a cost of forcing the end user to process and return the received list a bit more carefully.